force-app/main/default/lwc/serviceDeliveryRow/serviceDeliveryRow.js

```diff
@@ -123,6 +127,17 @@ export default class ServiceDeliveryRow extends LightningElement {
         }
     }
 
+    checkFieldExists() {
```

Comment:
Do we need this method here if the values are being passed in via API properties?  Appears that it can be removed.

Reply:
@AndersonTarren , Thank you for catching it. It needs to be removed. I will do that now.